src/components/RequestPanel/ConfirmRequest.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { Button, GU, Info, textStyle, useTheme } from '@aragon/ui'
+
+function ConfirmRequest({ descriptions, onStartRequest }) {
+  const theme = useTheme()
+
+  return (
+    <div>
+      <Info>
+        <h4
+          css={`
+            ${textStyle('label2')};
+            margin-bottom: ${0.5 * GU}px;
+          `}
+        >
+          Action{descriptions.length > 0 ? 's' : ''} to be triggered
+        </h4>
+        <div
+          css={`
+            color: ${theme.content};
+          `}
+        >
+          {descriptions.length > 1 ? (
```

Comment:
Would it be possible to see screenshots of each case? Or maybe we can jump on a call to demo this in a few scenarios quickly!

Reply:
Ah Yes! I will upload some screenshots and can do a quick demo in the standup tomorrow